lang/cs.json

```diff
@@ -113,15 +113,6 @@
   "WEAPON.SKILL_BONUS": "Dovednostní bonus",
   "WEAPON.FEATURE": "Vlastnosti",
 
-  "ITEM.ATTACK": "Útok",
```

Comment:
Careful of deleting this one. There is no equivalent in TypeThing

Reply:
Actually, it was used for the monster attack. See `chat/item.html` line 23. :)

Reply:
> Actually, it was used for the monster attack. See `chat/item.html` line 23. :)
> Needless to say `Monsterattack` is MUCH improved naming-wise.

Haven't checked. This is the only use of this string?

Reply:
Yes, it was the only place it was used.
After this change, there are no remaining uses of `ITEM.ATTACK`.